CMakeLists.txt

```diff
@@ -292,6 +280,24 @@ if(target MATCHES "win32")
   )
 endif()
 
+target_compile_definitions(
+  bare
+  PRIVATE
+    BARE_USE_SYSTEM_LOG=$<BOOL:${BARE_USE_SYSTEM_LOG}>
+)
+
+if(BARE_USE_SYSTEM_LOG)
+  if(NOT TARGET log)
+    add_subdirectory(vendor/liblog EXCLUDE_FROM_ALL)
+  endif()
+
+  target_link_libraries(
+    bare
+    PUBLIC
+      log_static
```

Comment:
That's the trickiest part, by switching `log_static` with `log`, the liblog methods aren't recognized, resulting in cmake `Undefined symbols` errors.